crypto/s2n_stream_cipher_rc4.c

```diff
@@ -23,9 +23,13 @@
 
 static uint8_t s2n_stream_cipher_rc4_available()
 {
-    return (EVP_rc4() ? 1 : 0);
```

Comment:
The old implementation implies that sometimes EVP_rc4 returns null, which indicates that the method exists but is not supported. Unless we're sure that's not valid behavior for any libcrypto, probably best to continue supporting that.

Reply:
Or I guess you could add a check that the method isn't null to your feature probe 🤔

Reply:
Fixed !